custom-wiki-links.ts

```diff
@@ -0,0 +1,98 @@
+// Custom remark plugin to handle wiki links with proper path resolution
+
+export function customWikiLinks() {
+  const searchDirs = ["zk", "zk/Register"];
```

Comment:
Only these two directories are searched, but wiki links may point to source files under other directories. Should search all directories recursively.